scripts/check-features.sh

```diff
@@ -9,7 +9,7 @@ echo "Checking all feature combinations with cargo-hack..."
 
 # Set environment variables to treat warnings as errors
 export RUSTFLAGS="-D warnings"
-export MIDEN_BUILD_LIBCORE_DOCS=1
+export MIDEN_BUILD_CORE_LIB_DOCS=1
```

Comment:
I would maybe rename this here (and in other relevant places) to just `MIDEN_BUILD_LIB_DOCS` to imply that this is for building library docs in general (not just core library docs).